src/pages/InsertJobs.tsx

```diff
@@ -0,0 +1,170 @@
+import { FormEvent, useState } from "react";
+import "../App.css";
+import JobCard from "../components/JobCard";
+
+interface JobData {
+  id: string;
+  title: string;
+  description: string;
+  jobType: "Estágio" | "Trainee" | "Júnior" | "";
+}
+
+function InsertJobs() {
+  const generateID = () => Math.floor(Date.now() * Math.random()).toString(36);
+
+  const [jobsList, setJobsList] = useState<JobData[]>([
+    {
+      id: generateID(),
+      title: "Desenvolvedor",
+      description:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.",
+      jobType: "Júnior",
+    },
+  ]);
+
+  const emptyJobData: JobData = {
+    id: generateID(),
+    title: "",
+    description: "",
+    jobType: "",
+  };
+
+  const [jobData, setJobData] = useState<JobData>(emptyJobData);
+
+  // FUNÇÕES
+  // VALIDAR VALORES DE UM OBJETO
+  function isValuesValid(object: any) {
```

Comment:
acho que o parâmetro "object" poderia trocar o nome que faça mais sentido tipo: "formData" | "payload" ...
ai você tipo como object.

Reply:
Na verdade essa função é pura, sendo assim ela sempre receberá um objeto, seja ele qual for, e retornará um booleano se pelo menos um dos valores de cada propriedade do mesmo é falsa. 
Portanto essa é a razão para a nomenclatura do parâmetro ser algo genérico como "object". Posteriormente irei adicionar o tipo object a esse parâmetro, pois é devido a esse código já fazer tempo que fora implementado, e na época estava escrevendo solo. 
Obrigado pelas considerações!